posts/2019-09-18-upcoming-docsrs-changes.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Upcoming docs.rs changes"
+author: The Rust Infrastructure Team
+---
+
+On September 42nd breaking changes will be deployed to the [docs.rs] build
+environment. [docs.rs] is a free service building and hosting documentation for
+all the crates published on [crates.io]. It's [open source][docsrs-source] and
+maintained by the [Rustdoc team][rustdoc-team].
+
+## What will change
+
+Builds will be executed inside the [rustops/crates-build-env] Docker image.
+That image contains a lot of system dependencies installed to ensure we can
+build as many crates as possible. It's already used by [Crater], and we added
+all the dependencies previously installed in the legacy build environment.
+
+To ensure we can continue operating the service in the future and to increase
+its reliability we also improved the sandbox builds are executed into, adding
+new limits:
+
+* Each platform will now have **15 minutes** to build its dependencies and
+  documentation.
+* **3 GB of RAM** will be available for the build.
+* Network access will be completly **disabled**.
+* Only the `target/` directory will be writable, and it will be purged after
+  each build.
+
+Finally, docs.rs will now use the latest nightly available when building
+crates.
```

Comment:
We should make a note to update this post after deploying the changes to link to the limits page here.